contracts/core/EntryPoint.sol

```diff
@@ -233,7 +233,7 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
             if (address(aggregator) != address(0)) {
                 // solhint-disable-next-line no-empty-blocks
                 try aggregator.validateSignatures(ops, opa.signature) {} catch {
-                    revert SignatureValidationFailed(address(aggregator));
+                    revert SignatureValidationFailed(totalOps + i, address(aggregator));
```

Comment:
opIndex is wrong here: aggregator failure is not attached to a specific UserOperation, but for a bulk of them.
aggregator should be unique, and thus reverting with an aggregator should be enough.
If you do want to give an index, then it should be the "aggIndex" which is just `i`

(Same index mistake with "AA96" above, which is also not a FailedOp - but there it is less critical, since it is just an "input sanitizer", on the ABI call from the bundler)

Reply:
@drortirosh could you please push a fix to both errors you've highlighted here?